pkgs/top-level/all-packages.nix

```diff
@@ -36625,7 +36625,6 @@ with pkgs;
   xdg-desktop-portal-gtk = callPackage ../development/libraries/xdg-desktop-portal-gtk { };
 
   xdg-desktop-portal-hyprland = callPackage ../applications/window-managers/hyprwm/xdg-desktop-portal-hyprland {
-    stdenv = gcc13Stdenv;
```

Comment:
need to add this back and the hyperlang one due to aarch64 still using gcc12 -- apologies.

Reply:
no problem, reverted.

Reply:
I think that the one for xdg-desktop-portal-hyprland still needs to get re-added. ofborg still hasn't gotten around to building the aarch64-linux, but seems like it will break.